packages/shared/providers.yaml

```diff
@@ -2296,6 +2296,21 @@ wave-accounting:
     proxy:
         base_url: https://gql.waveapps.com
     docs: https://docs.nango.dev/integrations/all/wave-accounting
+wealthbox:
+    categories:
+        - crm
+    auth_mode: OAUTH2
+    authorization_url: https://app.crmworkspace.com/oauth/authorize
+    token_url: https://app.crmworkspace.com/oauth/token
+    authorization_params:
+        response_type: code
+    token_params:
+        grant_type: authorization_code
+    refresh_params:
+        grant_type: refresh_token
+    proxy:
+        base_url: https://api.crmworkspace.com
```

Comment:
Very confusing from them to have a different hostname :D

Reply:
I actually noted that too. I had to triple check as well 😄.